python/cuml/tsa/arima.pyx

```diff
@@ -138,15 +138,15 @@ class ARIMA(Base):
 
     Attributes
     ----------
-    order : Tuple[int, int, int]
-        The ARIMA order (p, d, q) of the model
+    order : ARIMAOrder
+        The ARIMA order of the model (p, d, q, P, D, Q, s, k)
     seasonal_order: Tuple[int, int, int, int]
         The seasonal ARIMA order (P, D, Q, s) of the model
     intercept : bool or int
         Whether the model includes a constant trend mu
     d_y: device array
         Time series data on device
-    num_samples: int
+    n_obs: int
```

Comment:
good catch

Reply:
Just pulling in doc improvements from the other PR which are 0 risk. Louis caught this originally.